### PR TITLE
Refactor calculating skipped tests

### PR DESCRIPTION
**Pull request by karol-bisztyga**
Requested in https://github.com/software-mansion/protostar/pull/1254#discussion_r1046101744

Depends on #1254

### CODE REVIEW

**Comment by karol-bisztyga** [comment]
This is built on top of #1254, the new code here starts @[refactor](https://github.com/software-mansion/protostar/pull/1315/commits/b497d91be8e659fe92521d81dfc4350472d19647). For the clarity's sake, I'm going to wait until the base branch is merged before I mark this on as "ready for review".

File: protostar/testing/testing_summary.py
Comment (kasperski95) [inline protostar/testing/testing_summary.py:70]:
```suggestion
    def get_skipped_test_cases_count(self) -> int:
        return _calculate_skipped(
            broken_count=len(self.broken),
            failed_count=len(self.failed),
            passed_count=len(self.passed),
            total_count=self.test_collector_result.test_cases_count,
        )

    def get_skipped_test_suites_count(self) -> int:
```

**Review by kasperski95 — APPROVED** [review]

File: protostar/testing/testing_summary.py
Comment (karol-bisztyga) [reply protostar/testing/testing_summary.py]: [add _count suffixes](https://github.com/software-mansion/protostar/pull/1315/commits/7802924cdbd56369cdc4d3e4ad587d2ac99ca36b)

**Pull request merged**
